.circleci/config.yml

```diff
@@ -89,19 +73,9 @@ jobs:
           command: cd /home/circleci/hasadna/ && ./compile.sh build
```

Comment:
@oferb it matches the behaviour we currently have: https://github.com/google/startup-os/blob/d98b1eddc4989ef4f11d27d4914a4565df8cc9c6/.circleci/config.yml#L87-L91

Initially we did it because `bazel test` with no tests exits with [exit code 4](https://github.com/bazelbuild/bazel/blob/master/src/main/java/com/google/devtools/build/lib/util/ExitCode.java#L48). Currently, we could do it, but there's only `checkstyle` tests which cannot really be broken by `@startup_os` — thus, we are only building.

Reply:
Ok